### DISCUSSION TITLE
Templater user script functions

### DISCUSSION THREAD

**Discussion by MichelleLatham** [BryanJenksCommunity/FAQ#175] [Q&A] [answered]
Hello!
I'm new to obsidian and just found the incredible resources that Bryan has been providing. I've downloaded the templates and plugins he uses but there seem to be a lot of user specified templater scripts that I don't have and I'm not sure what they are. Is there anywhere I can see what these templater functions are and copy them?

**Comment by tallguyjenks** [comment, marked as answer]
my template vault doesnt use user scripts those are the ones that are external to the obsidian vault. i rely on none of those. the templater functions themselves can be seen in the templater documentation, if you're talking about the templater.js code then that is just javascript code interacting with the templater API
